core/modules/startup/keyboard.js

```diff
@@ -0,0 +1,97 @@
+/*\
+title: $:/core/modules/startup/keyboard.js
+type: application/javascript
+module-type: startup
+
+Keyboard shortcut handling
+
+\*/
+(function(){
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+// Export name and synchronous status
+exports.name = "global-keyboard";
+exports.platforms = ["browser"];
+exports.after = ["startup"];
+exports.synchronous = true;
+
+exports.startup = function() {
+	if($tw.browser) {
+		// Get existing shortcut-tiddlers
+		var shortcutTiddlers = $tw.wiki.filterTiddlers("[all[shadows+tiddlers]tag[$:/tags/KeyboardShortcut]!has[draft.of]]"),
+			shortcutKeysList = [],
+			shortcutActionList = [],
+			shortcutParsedList = [];
```

Comment:
It would be worth a brief comment explaining what these lists are for, and what goes in them (something like [this](https://github.com/Jermolene/TiddlyWiki5/blob/master/core/modules/syncer.js#L55-L59))

Reply:
I'll add comments here